### PR TITLE
docs: add documentation for benchmarking stream density

### PR DESCRIPTION
**Pull request by jim-wang-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)
- [ ] No Employer Confidential details are added

## What requirement is this design document for?

#**issue_number**

### CODE REVIEW

**Review by brian-intel — APPROVED** [review]
LGTM

File: docs_src/performance-tools/benchmark.md [hunk 1]
```diff
@@ -39,6 +40,19 @@ consumption, and so on.
     python benchmark.py --compose_file ../../use-cases/gst_capi/add_camera-simulator.yml --compose_file ../../use-cases/gst_capi/add_gst_capi_yolov5_ensemble.yml
     ```
 
+## Benchmark Stream Density of CV Pipelines
+
+Benchmarking a pipeline can also discover the maximum number of workloads or streams that can be ran in parallel for a given target FPS. This information is useful to determine the hardware required to achieve the desired performance for CV pipelines.
```
Comment (NeethuES-intel) [inline docs_src/performance-tools/benchmark.md:45]:
```suggestion
Benchmarking a pipeline can also discover the maximum number of workloads or streams that can be run in parallel for a given target FPS. This information is useful to determine the hardware required to achieve the desired performance for CV pipelines.
```

File: docs_src/performance-tools/benchmark.md [hunk 2]
```diff
@@ -39,6 +40,19 @@ consumption, and so on.
...
+
+To run the stream density functionality use `--target_fps` and/or `--density_increment` as inputs to the `benchmark.py` script:
```
Comment (NeethuES-intel) [inline docs_src/performance-tools/benchmark.md:47]: Can you add a note on what the input paramters are ? `--density_increment` is not clear, `--target_fps` is clear.

File: docs_src/performance-tools/benchmark.md [hunk 3]
```diff
@@ -39,6 +40,19 @@ consumption, and so on.
     python benchmark.py --compose_file ../../use-cases/gst_capi/add_camera-simulator.yml --compose_file ../../use-cases/gst_capi/add_gst_capi_yolov5_ensemble.yml
     ```
 
+## Benchmark Stream Density of CV Pipelines
```
Comment (NeethuES-intel) [inline docs_src/performance-tools/benchmark.md:43]:
```suggestion
## Benchmark Stream Density for CV Pipelines
```

File: docs_src/performance-tools/benchmark.md [hunk 4]
```diff
@@ -39,6 +40,25 @@ consumption, and so on.
...
+
+To run the stream density functionality use `--target_fps` and/or `--density_increment` as inputs to the `benchmark.py` script:
+
+   ```bash
+    python benchmark.py  --retail_use_case_root ../../retail-use-cases --target_fps 14.95 --density_increment 1 --init_duration 40   --compose_file ../../retail-use-cases/use-cases/grpc_python/docker-compose_grpc_python.yml
+   ```
+
+where the parameters:
+- `target_fps` is the given target frames per second (fps) to achive for maximum number of pipelines
+- `density_increment` is to configure the benchmark logic to increase the number of pipelines each time while trying to find out the maximum number of pipelines before reach the given target fps.
```
Comment (NeethuES-intel) [inline docs_src/performance-tools/benchmark.md:55]:
```suggestion
- `density_increment` is to configure the benchmark logic to increase the number of pipelines each time while trying to find out the maximum number of pipelines before reaching the given target fps.
```

**Review by NeethuES-intel — APPROVED** [review]
LGTM, Minor change.

**Pull request merged**